src/coredump/coredump.c

```diff
@@ -1304,6 +1312,7 @@ static int process_backtrace(int argc, char *argv[]) {
         context[CONTEXT_SIGNAL]    = argv[2 + CONTEXT_SIGNAL];
         context[CONTEXT_TIMESTAMP] = argv[2 + CONTEXT_TIMESTAMP];
         context[CONTEXT_RLIMIT]    = argv[2 + CONTEXT_RLIMIT];
+        context[CONTEXT_HOSTNAME]  = argv[2 + CONTEXT_HOSTNAME];
 
         n_allocated = 33 + COREDUMP_STORAGE_EXTERNAL;
```

Comment:
this needs to be increased by one, too, no?

Reply:
The same apply also here. The variable context is statically allocated on stack using the constant (enum) _CONTEXT_MAX which was increased by adding the enum value CONTEXT_HOSTNAME.

Reply:
same here, iovec[] is not context[]...